resources/views/components/manage-notifications.blade.php

```diff
@@ -168,11 +172,13 @@ class="box-border flex items-center justify-center w-5 h-5 text-white rounded cu
                     @endif
                 </span>
 
-                <button wire:click="applyFilter('')" type="button" class="flex items-center space-x-2 button-secondary">
-                    <x-ark-icon name="reset" />
+                @if (! ARKEcosystem\Hermes\Enums\NotificationFilterEnum::isAll($this->activeFilter))
+                    <button wire:click="applyFilter('')" type="button" class="flex items-center space-x-2 button-secondary whitespace-nowrap">
```

Comment:
make the placeholder maintain the same height on desktop when there is or isn't a button so it doesn't jitter on the screen when you switch between filtered/unfiltered